### ISSUE TITLE
minimega/miniccc: disconnects and then fails to reconnect

### ISSUE THREAD

**Issue by jcrussell** [sandia-minimega/minimega#776]
Lots of:

```
2016/11/29 16:44:05 ERROR heartbeat.go:73: heartbeat failed: write unix @->/tmp/minimega/0/fs/cc: write: broken pipe
```

**Comment by djfritz** [comment]
Likely fixed by #780 

@jcrussell can you confirm?

**Comment by jcrussell** [comment]
Wasn't able to reproduce with HEAD.
